package.json

```diff
@@ -238,6 +238,7 @@
     "semver": "^7.1.1",
     "shell-escape": "^0.2.0",
     "split": "^1.0.1",
+    "stream-to-promise": "^2.2.0",
```

Comment:
The cli actually uses this directly but was relying on a version pulled in via a dependency, I've installed it explicitly to avoid potential future surprises